superset/viz.py

```diff
@@ -144,15 +144,16 @@ def query_filters(self, is_having_filter=False):
         """Processes the filters for the query"""
         form_data = self.form_data
         # Building filters
+        flts = []
```

Comment:
Now that we have comprehensive filters, I would just pass them "as is" to the `Datasource.query` methods (druid and sqla). We may need to alter the code a bit there though. That means there's probably no more need for this `query_filters` method anymore.